package.json

```diff
@@ -6,7 +6,7 @@
     "node": "10.15.1"
   },
   "scripts": {
-    "test": "standard && jest",
+    "test": "standard --env jest && jest",
```

Comment:
Do we need to specify `--env jest` if we also have `/* eslint-env jest */` in our files?

Reply:
Well, caught I added that initially so that standard would pass while I was focusing on jest. All fixed by adding this standard setting to `package.json`